packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/Authorize/template.success.tsx

```diff
@@ -0,0 +1,312 @@
+import React, { useState } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { path } from 'ramda'
+import styled from 'styled-components'
+
+import { Button, Icon, Image, Text } from 'blockchain-info-components'
+import { fiatToString } from 'blockchain-wallet-v4/src/exchange/currency'
+import { FiatType } from 'blockchain-wallet-v4/src/types'
+import { FlyoutWrapper, Row, Title, Value } from 'components/Flyout'
+import { BankDWStepType } from 'data/types'
+
+import { Props as _P } from '.'
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+`
+const BackContainer = styled(Text)`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  font-weight: 600;
+  font-size: 20px;
+`
+const DropdownTitleRow = styled.div<{ isPaymentInformation?: boolean }>`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  cursor: pointer;
+  padding: ${props => (props.isPaymentInformation ? '0 40px' : 'auto')};
+  /* chevorn icon rotation */
+  > span:last-child {
+    size: 10px;
+    transition: transform 0.2s;
+    color: ${props => props.theme.grey600};
+    &.active {
+      transform: rotate(180deg);
+    }
+  }
+`
+const InfoTitle = styled(Title)`
+  font-weight: 600;
+  line-height: 1.5;
+  color: ${props => props.theme.grey900};
+`
+
+const InfoDropdown = styled.div`
+  max-height: 0;
+  margin-top: 0;
+  overflow: hidden;
+  transition: max-height, margin-top 0.3s;
+  &.isToggled {
+    max-height: 100%;
+    margin-top: 12px;
+  }
+`
+const InfoText = styled(Title)`
+  font-size: 14px;
+  font-weight: 500;
+  color: ${props => props.theme.grey600};
+  line-height: 1.5;
+`
+const Bottom = styled(FlyoutWrapper)`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  height: 100%;
+`
+const DropdownRow = styled(Row)<{ isPaymentInformation?: boolean }>`
+  padding: ${props => (props.isPaymentInformation ? '16px 0' : 'auto')};
+`
+
+const DropdownItem = ({ bodyText, isPaymentInformation, titleText }) => {
```

Comment:
Is this basically the same as simple buy flow authorize screen dropdown toggle? Maybe bump this up to a component directory so that you can share it across both features. Also this would make a perfect higher order component.